tests/load/test_databases.py

```diff
@@ -44,7 +44,10 @@ def test_timestream_write(benchmark_time: int, timestream_database_and_table: st
     df_cpu = df[df.measure_kind == "cpu_utilization"]
     df_memory = df[df.measure_kind == "memory_utilization"]
 
-    with ExecutionTimer("elapsed time of wr.timestream.write()") as timer:
+    with ExecutionTimer(
+        "elapsed time of wr.timestream.write()",
```

Comment:
I have suggestion to abstract this and avoid repetition/automate it.

In pytest there is this [request](https://stackoverflow.com/a/34732269/8178963) fixture which among other things returns the name of the test. So perhaps we can simply the ExecutionTimer so that it looks like:

```
def test_timestream_write(benchmark_time: int, timestream_database_and_table: str, request, cloudwatch_metric_data) -> None:
...
with ExecutionTimer(test_name=request.node.name, cloudwatch_metric_data=cloudwatch_metric_data) as timer:
     ...
```

and then inside the class we can handle creating both the string (i.e. `f"elapsed time of {self.test_name}"`) and manipulating the CW dict?